backend/ethereum/channel/funder.go

```diff
@@ -59,6 +62,12 @@ type Funder struct {
 var _ channel.Funder = (*Funder)(nil)
 
 // NewFunder creates a new ethereum funder.
+//
+// In addition to the `Fund` method required by the `Funder` interface, it also
+// providers two additional functions for convenience: RegisterAsset and
+// IsAssetRegistered.
+//
+// All the exported methods are thread-safe and can be invoked concurrently.
```

Comment:
This should go on type funder, I would say.

Reply:
Updated in this [force-push](https://github.com/hyperledger-labs/go-perun/compare/c5255fbe5f5453796953eb1c9e559a9565a7c2ea..4a12ab6d8062a3296b14d92a697dc705aec0b7e5).